public/css/presentation-style.css

```diff
@@ -1,33 +1,33 @@
-@import url("https://cdn.jsdelivr.net/npm/water.css@2/out/water.css");
 
-div.container{
+.container {
```

Comment:
Update this css to just use class names instead of keying off of elements.

Reply:
sounds good! is that in general better css practice?

Reply:
Generally, yes.